examples/publish_course_manual/publish_manual.sh

```diff
@@ -29,13 +29,15 @@ for app in jq md5sum curl echo base64 awk xxd file basename wc; do command -v "$
 
 #### Variables ####
 readonly BASE64=$( ( (echo test | base64 -w 0 > /dev/null 2>&1) && echo "base64 -w 0") || echo base64)
+
 readonly HTML_FILE='./ABC_training_course_manual.html'
 readonly COURSE_ID="REPLACE WITH COURSE_ID"
 readonly ATTACHMENT_DIR='./files'
 readonly COURSE_MANAGER_SUBDOMAIN="REPLACE WITH COURSE_MANAGER_SUBDOMAIN"
 readonly COURSE_MANAGER_HOST="skytap-portal.com"
 readonly API_TOKEN="REPLACE WITH API_TOKEN"
 readonly API_SECRET="REPLACE WITH API_SECRET"
+
```

Comment:
Any reason for the added lines in variables up here?

Reply:
Nope, sloppy copy/paste. fixed.